src/main/java/com/welcome/bot/controllers/UserController.java

```diff
@@ -57,18 +64,38 @@
     private UserSettingsRepository userSettingsRepository;
     @Autowired
     MessageSource messageSource;
+    @Autowired
```

Comment:
Use constructor level injection.

Reply:
I don't need constructor for Controller class because I will never use it's instance.

Reply:
> Every Time You Use Field Injection, a Unit Test Dies

https://dzone.com/articles/spring-di-patterns-the-good-the-bad-and-the-ugly